Testscripts/Linux/validate-intel-sgx-driver.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-intel-sgx-driver.sh
+# Description:
+#    Check the existence of Intel sgx driver.
+#    Install Open-Enclave and build&run Samples
+# Supported Distros:
+#    Ubuntu
+#######################################################################
+
+set -e
+set -x
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" >state.txt
+    exit 1
+}
+
+UtilsInit
+
+sudo apt-get update
+sudo apt-get -y upgrade
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-7 main" | sudo tee /etc/apt/sources.list.d/llvm-toolchain-bionic-7.list
+wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+
+echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/18.04/prod bionic main" | sudo tee /etc/apt/sources.list.d/msprod.list
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-get update
+
+LogMsg "----- Checking sgx driver -----"
+if ! modinfo intel_sgx; then
+    LogErr "modinfo intel_sgx failed"
+    SetTestStateFailed
+    exit 1
+fi
+if [ "$(stat -c %a /dev/sgx)" == 600 ]; then
+    LogMsg "/dev/sgx permission is incorrect: 0600, temporarily fix this issue by chmod 666"
+    sudo chmod 666 /dev/sgx
+fi
+
+LogMsg "----- Install Open Enclave packages and dependencies -----"
+sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
+
+SetTestStateRunning
+LogMsg "----- Runing Samples Tests -----"
+cd ~
+cp -r /opt/openenclave/share/openenclave/samples/ ~/samples
+source /opt/openenclave/share/openenclave/openenclaverc
+SAMPLES=$(find ./samples/* -maxdepth 0 -type d)
+ALL_PASS=true
```

Comment:
better to set false by default.

Reply:
I see you resolved this comment without any reply or change. Does it mean you would not make a change? In common software programming, the default value is 'false' in many programming language. Can you share your idea?